src/main/java/com/polyscripts/contactManagement/Services/ContactService.java

```diff
@@ -0,0 +1,34 @@
+package com.polyscripts.contactManagement.Services;
+
+import com.polyscripts.contactManagement.models.Contact;
+import com.polyscripts.contactManagement.repos.ContactEmployeeRepo;
+import com.polyscripts.contactManagement.repos.ContactFreelanceRepo;
+import com.polyscripts.contactManagement.repos.ContactRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ContactService  {
+
+    private final ContactRepo contactRepo;
```

Comment:
there is @Autowired annotation, it is always good to use spring boot annotations :)